CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ add_library(ViveLoaderLib STATIC
     VRSettings.h)
 target_link_libraries(ViveLoaderLib
     PUBLIC
-    OpenVRDriver osvr::osvrUtil linkable_into_dll
+    OpenVRDriver osvr::osvrUtil linkable_into_dll ${CMAKE_DL_LIBS}
```

Comment:
`$CMAKE_DL_LIBS` is set to an empty string on platforms that don't need to link to a library for `dlopen()`/`dlclose()`.